transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerDomainSocketChannel.java

```diff
@@ -82,9 +82,7 @@ protected void doClose() throws Exception {
                 // Delete the socket file if possible.
                 File socketFile = new File(local.path());
                 boolean success = socketFile.delete();
-                if (!success && logger.isDebugEnabled()) {
-                    logger.debug("Failed to delete a domain socket file: {}", local.path());
-                }
+                logger.debug("Delete a domain socket file [sucess:{}]: {}", success, local);
```

Comment:
Similar to other inline comment, not clear why the behaviour should be changed here.

Reply:
Please only log if not successful..